### ISSUE TITLE
Shared calendar, after the display name is missing a blank

### ISSUE THREAD

**Issue by prsnbrg** [owncloud/calendar#905]
### Steps to reproduce
1. I use the german version of oC
2. Shared a calender with a url
3. after open this url I see on the left site the display name from the sharing user

### Expected behaviour
Tell us what should happen
after the display name on the left site a blank.

### Actual behaviour
Tell us what happens instead
Shared calendar, after the display name is missing a blank

### Server configuration

**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**ownCloud version:** (see your admin page)
oC 10.0.7.2
**Calendar version:** (see the apps page)
1.5.7
**Updated from an older installed version or fresh install:**

**Signing status (ownCloud 9.0 and above):**

```
Login as admin user into your cloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your instance's installation folder
```

**ownCloud configuration:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your instance's installation folder

or 

Insert your config.php content here
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

#### LDAP configuration (delete this part if not used)

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your instance's installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Be sure to replace sensitive data as the name/IP-address of your LDAP server or groups.
```

### Client configuration
**Browser:**

**Operating system:**

**CalDAV-clients:**

#### Web server error log
```
Insert your webserver log here
```

#### Log file (data/owncloud.log)
```
Insert your ownCloud.log file here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by Julian1998** [comment]
Hi ;)

Ugh! :D Do you want to fix that? :)

**Comment by prsnbrg** [comment]
Thanks for the possibility, but I am a completely newbie on the github development and I don't have at the moment, to work on this.

**Comment by Julian1998** [comment]
Alright :)

@DeepDiver1975 I also wondered about that missing blank?

Is it on purpose because WebDav has some problems with parsing blanks?

**Comment by DeepDiver1975** [comment]
So you want to have Calendar (user) instead of Calendar(user) - right?

add the space in here https://github.com/owncloud/core/blob/8ed293b5d237e853789abdd5104f9c4e148c20db/apps/dav/lib/CalDAV/CalDavBackend.php#L240

**Comment by DeepDiver1975** [comment]
and here for addressbooks: https://github.com/owncloud/core/blob/8ed293b5d237e853789abdd5104f9c4e148c20db/apps/dav/lib/CardDAV/CardDavBackend.php#L179

**Comment by prsnbrg** [comment]
Yes this is right.

**Comment by DeepDiver1975** [comment]
merged into core master - backport to stable10 is open - will be released with 10.0.0 or 10.0.10
